src/datachain/lib/dc.py

```diff
@@ -2418,6 +2419,7 @@ def export_files(
         signal="file",
         placement: FileExportPlacement = "fullpath",
         use_cache: bool = True,
+        link_type: Literal["copy", "symlink"] = "copy",
```

Comment:
can we update / add docs while we do this (or as a followup)

Reply:
done in 2d9547f